### DISCUSSION TITLE
Managing PRs - using Draft status

### DISCUSSION THREAD

**Discussion by p-bakker** [mozilla/rhino#1616] [General]
Just a quick notification to all: I've started to convert some PRs to draft status, so the backlog of active PRs that are actually to be reviewed and considered for merging is smaller and thus more manageable. 

Hope this helps the maintainers to better manage things. If it doesn't, we'll revert course

See https://github.com/mozilla/rhino/pull/1384#issuecomment-1703018239 for the request from @gbrail to do this

At the moment we're below 20 non-DRAFT PRs, but still some quite old ones that I need to weed through

**Comment by gbrail** [comment]
Yes -- there is a "bottom 2/3" of PRs that are really old and I can't see a
path to merging any of them -- either I don't have the expertise in some of
the 10 different types of Java interop that would be needed to understand
if they'll break things, or they're just really old and need an update. Any
ideas how to address those?

**Reply by p-bakker** [reply]
Lets start with the non-interop ones: 
- #119 & #85 any comments on their merits? Yes they need a rebase, but the changes make sense to me, just not sure what else besides a rebase it'll take to get them merged
- #354 Can you check my comment on that one? I think we can close that one as Not Planned

Then for the Java Interop ones: 
- #601 Is there still merit to this one? I know you wanted to get rid of synchronization? If there's still merit, the MR needs to be updated
- #619 This one seems ready for merge, if rebased? All your comments seem to have been addresses I think
- #827 Seems to still have merit and @rPraml is still active, so I think we can work on moving this forward
- #1085 & #1541 The merit of both these is still being discussed (albeit on a slow schedule :-) )

**Comment by gbrail** [comment]
And in addition -- I have worked on projects that have a "waiting" status,
so that maintainers / reviewers can basically flag, "I'm not looking at
this until you answer my last question" kinds of things. They also nag the
person who should answer the question, and nag maintainers if they don't
look at something that they are not waiting for information on. Not sure
how much of that is default GitHub and how much is an add-on. Anyone else
have experience with that?

**Reply by p-bakker** [reply]
Got a pointer to a project that uses such a setup? Couldn't find any reference to something like that

**Reply by tonygermano** [reply]
Possibly a bot that changes who to nag (author or maintainers) based on the presence or absence of a tag on the PR?
